pkg/kubectl/resource_printer.go

```diff
@@ -145,6 +145,9 @@ type ResourcePrinter interface {
 	// Print receives a runtime object, formats it and prints it to a writer.
 	PrintObj(runtime.Object, io.Writer) error
 	HandledResources() []string
+	//Can be used to print out warning/clarifications if needed
+	//after all objects were printed
+	FinishPrint(io.Writer) error
```

Comment:
Since this is only ever used by `HumanReadablePrinter`, it should not be added to the interface. Suggest doing the check for hidden pods at the end of [`HumanReadablePrinter.PrintObj`](https://github.com/vefimova/kubernetes/blob/ec2a786d826ca6d97d5cf4f856a08bcdfc1313a0/pkg/kubectl/resource_printer.go#L2049), and printing the warning there.